src/change/sdp.rs

```diff
@@ -429,6 +429,47 @@ pub struct SdpPendingOffer {
     changes: Changes,
 }
 
+impl SdpPendingOffer {
+    /// Retry this `SdpPendingOffer`.
+    ///
+    /// This is useful for this sequence of events.
+    ///
+    /// 1. str0m creates an OFFER with some changes (this results in `SdpPendingOffer`).
+    /// 2. Client creates an OFFER with other changes.
```

Comment:
Client in this case means the other peer? Maybe use Alice and Bob :)

Reply:
Other peer is good. Not sure I'm ready for Alice and Bob :)